### PR TITLE
[RDY] Shortern the win fax messages to fit in the fax screen

### LINKED ISSUE #1663: Demo game win message has text overlap
#### Describe the issue
Seems the win message for the demo is too big for our fax. As a result, it encroaches on the choices section.

#### Expected Behavior
Do not overlap the text

#### System Information
**CorsixTH Version:** 0.64

**Operating System:** W10

**Theme Hospital Version:** Demo

### Additional Info
image

### PR DESCRIPTION
**Pull request by tobylane**
The fax message overruns won't overlap with the reply options. Fixes #1663. Changes in bold. Translators should be pointed to the original wording?

Before
> letter[9][2] = "You have proved yourself to be the best hospital administrator in medicine's long and chequered history. Such a momentous achievement cannot go unrewarded, so we would like to offer you the post of Supreme Chief of All Hospitals. **This is an honorary job,** and comes with a salary of $%d. You will be given a tickertape parade, and people will show their appreciation wherever you go.//"
letter[10][2] = "Congratulations on successfully running every hospital we assigned you to. Such a superb performance qualifies you for the freedom of all the world's cities. You are to be given a pension of $%d, **plus a limousine**, and all we ask is that you travel **from city to city, meeting your adoring public** and promoting the work of all hospitals everywhere.//"
letter[11][2] = "Your career has been exemplary, and you are an inspiration to all of us. Thank you for running so many hospitals **and doing so well in every job**. We would like to grant you a lifetime salary of $%d, and would ask simply that you travel by official open-topped car from city to city, giving lectures about how you achieved so much so fast.//"
letter[11][3] = "You are an example to every **right-thinking** person, and without exception, everybody in the world regards you as a supreme asset.//"
letter[12][2] = "Your successful career as the best hospital administrator since Moses is nearing an end. However, such has been your impact on the **cosy world of medicine, the Ministry** would like to offer you a salary of $%d simply to appear on our behalf, opening fetes, launching ships, and doing chat shows. **The whole world is clamouring for you, and** it would be great PR for us all!//"

After
> letter[9][2] = "You have proved yourself to be the best hospital administrator in medicine's long and chequered history. Such a momentous achievement cannot go unrewarded, so we would like to offer you the **honorary** post of Supreme Chief of All Hospitals. This comes with a salary of $%d. You will be given a tickertape parade, and people will show their appreciation wherever you go.//"
letter[10][2] = "Congratulations on successfully running every hospital we assigned you to. Such a superb performance qualifies you for the freedom of all the world's cities. You are to be given a pension of $%d, and all we ask is that you travel, for free, **around the nation**, promoting the work of all hospitals to your adoring public.//"
letter[11][2] = "Your career has been exemplary, and you are an inspiration to all of us. Thank you for running so many hospitals **so well**. We would like to grant you a lifetime salary of $%d, and would ask simply that you travel by official open-topped car from city to city, giving lectures about how you achieved so much so fast.//"
letter[11][3] = "You are an example to every **wise** person, and without exception, everybody in the world regards you as a supreme asset.//"
letter[12][2] = "Your successful career as the best hospital administrator since Moses is nearing an end. However, such has been your impact on the **nation, the PM** would like to offer you a salary of $%d simply to appear on our behalf, opening fetes, launching ships, and doing chat shows. It would be great PR for us all!//"

### CODE REVIEW

File: CorsixTH/Lua/languages/english.lua [hunk 1]
```diff
@@ -104,8 +104,13 @@ multiplayer.everyone_failed = "Everyone failed to satisfy that last objective. S
...
+letter[10][2] = "Congratulations on successfully running every hospital we assigned you to. Such a superb performance qualifies you for the freedom of all the world's cities. You are to be given a pension of $%d, and all we ask is that you travel, for free, around the nation, promoting the work of all hospitals to your adoring public.//"
+letter[11][2] = "Your career has been exemplary, and you are an inspiration to all of us. Thank you for running so many hospitals so well. We would like to grant you a lifetime salary of $%d, and would ask simply that you travel by official open-topped car from city to city, giving lectures about how you achieved so much so fast.//"
+letter[11][3] = "You are an example to every wise person, and without exception, everybody in the world regards you as a supreme asset.//"
+letter[12][2] = "Your successful career as the best hospital administrator since Moses is nearing an end. However, such has been your impact on the nation, the PM would like to offer you a salary of $%d simply to appear on our behalf, opening fetes, launching ships, and doing chat shows. It would be great PR for us all!//"
```
Comment (lewri) [inline CorsixTH/Lua/languages/english.lua:112]: Changing from "Ministry" to "PM" is quite the difference. Plus PM is acronym.

**Review by lewri — COMMENTED** [review]
If we plan (I assume) to implement the letter video of the original we might want to revert these after? Though I think that has stalled currently.

Otherwise the change looks okay with note.

**Comment by TheCycoONE** [comment]
@lewri Stalled in the sense that no one has ever worked on it at all to the best of my knowledge - yes.

File: CorsixTH/Lua/languages/english.lua (hunk 1 above)
Comment (tobylane) [reply CorsixTH/Lua/languages/english.lua:112]: A few characters could be saved elsewhere for Ministry, eg "nearing an end. **Befitting your impact on national medicine**, the Ministry would like to offer"

**Review by lewri — APPROVED** [review]

**Pull request merged**